src/Tribe/Main.php

```diff
@@ -1012,6 +1014,11 @@ public function init() {
 
 			$this->default_values                             = apply_filters( 'tribe_events_default_value_strategy', new Tribe__Events__Default_Values() );
 
+			/* Deprecated 4.0 */
```

Comment:
I just grouped these and moved them here to make it easier to remove them together.